pkg/configuration/parameters.go

```diff
@@ -168,8 +168,12 @@ func (param *JiraAPIResourceParameters) Parse() {
 		param.Meta.parsed = flag.Parsed()
 	}
 
-	param.initializeContext(contextString)
-	param.initializeIssueList(issueListString)
+	param.InitializeAndValidatePostParse(contextString, issueListString)
+}
+
+func (param *JiraAPIResourceParameters) InitializeAndValidatePostParse(context, issueList *string) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported method JiraAPIResourceParameters.InitializeAndValidatePostParse should have comment or be unexported](https://app.codacy.com/manual/TurnsCoffeeIntoScripts/jira-api-resource/pullRequest?prid=5035521)